environment/index.js

```diff
@@ -1,48 +1,74 @@
 
-const { program } = require('commander');
+const {program} = require('commander');
 
-const { CleanCommand } = require('./commands/clean');
-const { PrepareCommand } = require('./commands/prepare');
-const { StartCommand } = require('./commands/start');
-const { TestCommand } = require('./commands/test');
-const { TransferFunETH2NEAR } = require('./commands/transfer-fun-eth2near');
-const { InitETHTestContracts } = require('./commands/init-eth-test-contracts');
-const { InitNEARTestContracts } = require('./commands/init-near-test-contracts');
+const {CleanCommand} = require('./commands/clean');
+const {PrepareCommand} = require('./commands/prepare');
+const {StartCommand} = require('./commands/start');
+const {TestCommand} = require('./commands/test');
+const {TransferFunETH2NEAR} = require('./commands/transfer-fun-eth2near');
+const {InitETHTestContracts} = require('./commands/init-eth-test-contracts');
+const {InitNEARTestContracts} = require('./commands/init-near-test-contracts');
 
 program.version('0.1.0');
 
 program.command('clean').action(CleanCommand.execute);
 
-program.command('start <service>').action(StartCommand.execute);
+program.command('start')
```

Comment:
I would split this command into four commands:
* One that starts local near node;
* One that starts local Ganache;
* One that starts ethrelay;
* One that starts nearrelay.